src/local-development/create-remote-component.ts

```diff
@@ -42,6 +43,8 @@ export async function createRemoteAppComponent(opt: {
 		const axiosConfig: AxiosRequestConfig = {
 			headers: {
 				Authorization: 'Token ' + opt.origin.apikey,
+				'x-imt-vsce-localmode': true,
```

Comment:
I like the prefix `x-imt-vsce-`. Makes more sense that previously used `x-imt-apps-sdk`... Good idea.